#断言可以中断程序，对于调试很重要。
# 当assert后条件为真时，程序运行，当条件为假时，程序中断并输出错误信息。

x=10
assert x>0, 'Error: x<=0' 
print("第一个assert啥都没发生")

#经过一系列运算后
x*=-5

#我们要确保x还是正数
assert x>0, 'Error: x<=0' #结果不是正数

print("该行不会执行")